src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,15 +2800,16 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 4 failures in `PowerShell-CI-windows`
Tests for (error, warning, etc) action preference.Setting ErrorActionPreference to stop prevents user from getting the error exception.$err.Count
```powershell
Expected 1, but got 2.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Scripting\ActionPreference.Tests.ps1: line 79
79:         It '$err.Count' { $err.Count | Should -Be 1 }
```
Test PSDesiredStateConfiguration.Get-DscResource - Class base Resources.should be able to get class resource - XmlFileContentResource from XmlContentDsc - Good case
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\PSDesiredStateConfiguration\PSDesiredStateConfiguration.Tests.ps1: line 323
323:             $resource | Should -Not -BeNullOrEmpty
```
Test PSDesiredStateConfiguration.Get-DscResource - Class base Resources.should be able to get class resource - XmlFileContentResource from xmlcontentdsc - Module Name case mismatch
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\PSDesiredStateConfiguration\PSDesiredStateConfiguration.Tests.ps1: line 323
323:             $resource | Should -Not -BeNullOrEmpty
```
Test PSDesiredStateConfiguration.Get-DscResource - Class base Resources.should be able to get class resource - xmlfilecontentresource from XmlContentDsc - Resource name case mismatch
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\PSDesiredStateConfiguration\PSDesiredStateConfiguration.Tests.ps1: line 323
323:             $resource | Should -Not -BeNullOrEmpty
```